docs/source/tuning_strategies.md

```diff
@@ -145,37 +145,74 @@ conf = PostTrainingQuantConfig(
 
 ### Design
 
-The `Basic` strategy is designed for quantizing most models. There are six stages executed by `Basic` strategy sequentially, and the tuning process ends once the condition meets the exit policy. 
+The `Basic` strategy is designed for quantizing most models. There are several stages executed by `Basic` strategy sequentially, and the tuning process ends once the condition meets the exit policy.  The diagram below illustrates each stage, accompanied by additional details provided for each annotated step.
+
+
+
+```mermaid
```

Comment:
Pls refer to the rendered version for better UX. (https://github.com/intel/neural-compressor/blob/ly/block_doc/docs/source/tuning_strategies.md#design-2)